sympy/utilities/autowrap.py

```diff
@@ -65,7 +65,6 @@
 
 """
 
-from __future__ import print_function, division
 
```

Comment:
Remove this newline from here as well as elsewhere. Keep only one newline between docstring and imports.

Reply:
An automatic code formatter like `black` would be awesome for things like this

Reply:
Fixed all of those manually :-)